functions/go/create-setters/createsetters/create_setters.go

```diff
@@ -354,5 +358,13 @@ func Decode(rn *yaml.RNode, fcd *CreateSetters) error {
 
 	// sorts all the Setters
```

Comment:
Separate the code after this into a different method like preProcessScalarSetters. 

This should be invoked at the beginning in Filter method.